lib/src/presentations/home/screen/home_page.dart

```diff
@@ -0,0 +1,221 @@
+      children: [
+        Text(
+          AppLocal.text.home_page_recent_job_list,
+          style: AppStyles.boldText.copyWith(fontSize: 16),
+        ),
+        const SizedBox(height: 16),
+        ListView.separated(
+          physics: const NeverScrollableScrollPhysics(),
+          shrinkWrap: true,
+          itemBuilder: (context, index) {
+            return CustomJobCard(
+              button: IconButton(
+                icon: SvgPicture.asset(AppImages.save),
+                onPressed: () {},
+              ),
+              onTap: () {},
+              isShowApply: true,
+              onApply: () {},
+            );
+          },
+          separatorBuilder: (context, index) => const SizedBox(height: 10),
+          itemCount: 10,
+        )
+      ],
+    );
+  }
+
+  PreferredSizeWidget _buildAppBar({
+    required double width,
+    required VoidCallback onTap,
+  }) {
+    final user = PrefsUtils.getUserInfo();
+    return PreferredSize(
+      preferredSize: Size(width, 90),
+      child: SafeArea(
+        child: Padding(
+          padding: const EdgeInsets.symmetric(
+            horizontal: AppDimens.smallPadding,
+            vertical: 10,
+          ),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Text(
+                AppLocal.text.home_page_hello(user?.name ?? ""),
+                style: AppStyles.boldTextNightBlue.copyWith(fontSize: 22),
+              ),
+              GestureDetector(
+                onTap: onTap,
+                child: ClipOval(
+                  child: user!.avatar.isEmpty
+                      ? SvgPicture.asset(AppImages.logo, height: 40, width: 40)
+                      : CachedNetworkImage(
+                          imageUrl: user.avatar,
+                          placeholder: (_, __) => const SizedBox(),
+                          errorWidget: (_, __, ___) =>
+                              const Icon(Icons.warning),
+                          height: 40,
+                          width: 40,
+                        ),
+                ),
+              )
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildFindYourJob(BuildContext context, {required double width}) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          AppLocal.text.home_page_find_your_job,
+          style: AppStyles.boldText.copyWith(fontSize: 16),
+        ),
+        const SizedBox(height: 20),
+        Row(
+          children: [
+            _buildRemoteJobWidget(
+              width,
+              onTap: () {},
+            ),
+            const SizedBox(width: 20),
+            Column(
+              children: [
+                _buildJobContainer(
+                  width: width,
+                  title: AppLocal.text.home_page_full_time,
+                  quantity: "66.8k",
```

Comment:
Remove hard code

Reply:
I write hard text to test UI. I wrote a comment